src/v1/providers/firestore.ts

```diff
@@ -52,38 +49,32 @@ export function document<Path extends string>(path: Path) {
   return _documentWithOptions(path, {});
 }
 
-/** @hidden */
 // Multiple namespaces are not yet supported by Firestore.
 export function namespace(namespace: string) {
```

Comment:
If not hidden, can we try to documen this?

We can also consult w/ @egilmorez when he's back to document this section if you aren't sure how to document this.

Reply:
I don't have enough context on this..

Reply:
Let's leave it blank for now.

Reply:
kko!